submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+/* NAV START */
+.container {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 340px;
+    position: absolute;
+    right: 10px;
+    top: -10px;
+}
+
+.element {
+    margin: 5px;
+}
+
+.tiel {
+    background-image: url(img/tile.png);
+    width: 32px;
+    height: 32px;
+}
+
+.icon {
+    width: 32px;
+    height: 32px;
+    border-radius: 50%;
+    background-image: url(img/account.png);
+}
+
+.element a {
+    text-decoration: none;
+    color: #111;
+    font-size: 1em;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+li {
+    list-style: none;
+    text-transform: capitalize;
+}
+
+/* NAV END */
+
+/* popup-ul START*/
+.popup {
+    display: flex;
+    flex-wrap: wrap;
+    width: 320px;
+    height: 500px;
+    border: #ccc solid 1px;
+    border-radius: 2px;
+    position: relative;
+    top: 10px;
+    right: 250px;
+    display: none;
+    overflow-y: auto;
+}
+
+.popup-ul,
+.more-ul {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 28px;
+    padding-bottom: 0px;
+}
+
+.popup-li {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 2px;
+    width: 65px;
+    margin: 8px;
+    border: 1px solid transparent;
+}
+
+.popup-li:hover {
+    border: 1px solid #ddd;
+}
+
+.popup-a:focus {
+    border: 1px solid rgba(125, 125, 255, 0.5);
```

Comment:
This border creates a bug, when I focused icon, your layout jumping, because height changes. Why we need two borders, from li tag hover, and a tag focus? You can check, how google do this :)

hint: add icon name into tag a

Reply:
@CuteShaun this bug is fixed